tests/view_tests/tests/test_debug.py

```diff
@@ -29,9 +28,7 @@
     sensitive_args_function_caller, sensitive_kwargs_function_caller,
     sensitive_method_view, sensitive_view,
 )
-
-if six.PY3:
-    from .py3_test_debug import Py3ExceptionReporterTests  # NOQA
+from .py3_test_debug import Py3ExceptionReporterTests  # NOQA
```

Comment:
Remove `py3_test_debug.py` from setup.cfg (a better solution here is to move the test into this file).